packages/evm/src/evm.ts

```diff
@@ -1012,6 +1012,19 @@ export class EVM implements EVMInterface {
         message.isCompiled = true
       } else {
         message.code = await this.stateManager.getCode(message.codeAddress)
+
+        // EIP-7702 delegation check
+        if (
+          this.common.isActivatedEIP(7702) &&
+          equalsBytes(message.code.slice(0, 3), new Uint8Array([0xef, 0x01, 0x00]))
```

Comment:
Could we make a constant out of `0xef0100` and call it `DELEGATION_DESIGNATOR` or whatever is the best way to refer to this instead of a magic number?